src/Nethermind/Nethermind.Network.Stats/Model/NodeStatsEventType.cs

```diff
@@ -49,6 +51,11 @@ public enum NodeStatsEventType
         SyncFailed,
         SyncCompleted,
 
-        Disconnect
+        LocalDisconnectDelay,
+        RemoteDisconnectDelay,
```

Comment:
Do we want to add counters to: https://github.com/NethermindEth/nethermind/blob/6c507f3e73307c527e72275419c6c1db14c344cd/src/Nethermind/Nethermind.Network/PeerManager.cs#L507-L519 ?

Reply:
Not really, the counters are too much IMO. And no decrement that I can find.